docs/devices/PTM_215Z.md

```diff
@@ -20,7 +20,7 @@ pageClass: device-page
 | Description | Pushbutton transmitter module |
 | Exposes | action, linkquality |
 | Picture | ![EnOcean PTM 215Z](https://www.zigbee2mqtt.io/images/devices/PTM-215Z.png) |
-| White-label | Niko 91004, NodOn CWS-4-1-01_HUE, Vimar 03906, Sunricher SR-ZGP2801K4-FOH-E, LED-Trading 9125, Feller 4120.2.S.FMI.61 |
+| White-label | Niko 91004, NodOn CWS-4-1-01_HUE, Vimar 03906, Sunricher SR-ZGP2801K4-FOH-E, LED-Trading 9125, Feller 4120.2.S.FMI.61, Namron ZigBee FOH Green Bryter K4 (4512727) |
```

Comment:
This is generated from https://github.com/Koenkk/zigbee-herdsman-converters/blob/cc8bff02f706007f990126310fee13f1e64781f4/src/devices/enocean.ts#L16, could you update that also?

Reply:
I've added a PR for the change If it's generated from the file you, do you want me to remove it from here?

Reply:
It's fine to leave it here for now, thanks for the PR!